Parttime Platforms Spring 2021/index.md

```diff
@@ -0,0 +1,167 @@
+# Scientific Programming 2
+
+## Syllabus<br><small>Parttime Platforms Spring 2021</small>
+
+Welcome to this programming course! In the weeks ahead, you'll use the programming language Python while learning to solve scientific problems from several fields of science.
+
+## Staff
+
+Simon Pauw
+
+Natasja Wezel
+
+Iris Luden
+
+Martijn Stegeman
+
+Björn Out
+
+If you have practical matters that you would like to discuss, always send an e-mail to the staff via <scientific@mprog.nl>. We will answer within a couple of days, if not hours.
+
+## What to do
+
+Your entry to the course is the sidebar, where you can leaf through all modules (levels) that you have to complete.
+
+## Passing the course
+
+You will take the three modules of Scientific Programming 2 for the first half of the platforms course. The modules are graded pass/fail only: you must submit fully working solutions for each module. The material of these three modules will be part of the oral exam at the end of the course.
+
+## Asking questions
+
+In this course you'll mostly work on assignments independently. But you're not on your own! We're here to help. There are two ways you can get help:
+
+* Online lab-sessions: We have created an online classroom on [wonder.me](https://www.wonder.me/r?id=8b9b7728-3e68-4924-ad4b-8dca2c2c8c70). In this classroom you can get help from us (and from your fellow students). We will be available at the office-hours mentioned here below. But you can log into the classroom at any time. So also outside of the office hours, you can use the classroom to meet up with other students.
+* You can also ask questions on [Ed](https://edstem.org/us/courses/4337/), an online discussion platform. You can use this to sign up link: [sign up for the Scientific Programming forum](https://edstem.org/us/join/za4ucr). Try to formulate your question clearly. Use code fragments to illustrate the problem. But, *never copy your entire code* here (this would make it too tempting for your fellow students to copy your code).
+
+There are four moments in the week that there is help available on wonder.me:
+
+|               | Mon | Tue | Wed | Thu | Fri |
+|---------------|:---:|:---:|:---:|:---:|:---:|
+| 09:00 - 12:00 |     |  ✓  |     |  ✓  |     |  
+| 14:00 - 17:00 |  ✓  |     |     |     |  ✓  |
+
+## Deadlines
+
```

Comment:
Note: for you, this is the first half of the course. For the second half, there will be a separate website with separate deadlines.